### ISSUE TITLE
bug: CocoaPods compatibility error with "GoogleUtilities/Environment"

### ISSUE THREAD

**Issue by ghonche** [robingenz/capacitor-firebase-performance#19]
**Plugin version:**
1.2.0

**Platform(s):**
ios

**Current behavior:**
After installing @robingenz/capacitor-firebase-performance , running `npx cap sync` generates this error:

> ✖ Updating iOS native dependencies with pod install - failed!
✖ update ios - failed!
[error] Analyzing dependencies
        [!] CocoaPods could not find compatible versions for pod "GoogleUtilities/Environment":
        In Podfile:
        CordovaPluginsStatic (from `../capacitor-cordova-ios-plugins`) was resolved to 3.3.3, which depends on
        GoogleUtilities ( ~> 7.2.2) was resolved to 7.2.2, which depends on
        GoogleUtilities/Environment (= 7.2.2)
        RobingenzCapacitorFirebasePerformance (from `../../node_modules/@robingenz/capacitor-firebase-performance`) was
        resolved to 1.3.0, which depends on
        Firebase/Performance (= 8.4.0) was resolved to 8.4.0, which depends on
        FirebasePerformance ( ~> 8.4.0) was resolved to 8.4.0, which depends on
        GoogleUtilities/Environment ( ~> 7.4)
        

**Steps to reproduce:**
On a capacitor project run:

> npm install @capacitor/ios

> npx cap add ios

> npm i @robingenz/capacitor-firebase-performance

> npx cap sync

**Capacitor doctor:**

```
💊   Capacitor Doctor  💊 
Latest Dependencies:
  @capacitor/cli: 3.4.0
  @capacitor/core: 3.4.0
  @capacitor/android: 3.4.0
  @capacitor/ios: 3.4.0
Installed Dependencies:
  @capacitor/cli: 3.3.3
  @capacitor/core: 3.3.3
  @capacitor/ios: 3.3.4
  @capacitor/android: 3.3.3
[success] iOS looking great! 👌
[success] Android looking great! 👌
```

**Comment by robingenz** [comment]
Hi @ghonche,
thank you for your request.
I cannot reproduce the issue with my demo app: https://github.com/robingenz/capacitor-firebase-plugin-demo

> CordovaPluginsStatic (from ../capacitor-cordova-ios-plugins) was resolved to 3.3.3, which depends on GoogleUtilities ( ~> 7.2.2) was resolved to 7.2.2, which depends on GoogleUtilities/Environment (= 7.2.2)

I assume from this output that you are using a different Firebase plugin that uses an older version of `GoogleUtilities/Environment`.
To be sure, I need a Minimal, Reproducible Example (see [How to create a Minimal, Reproducible Example](https://stackoverflow.com/help/minimal-reproducible-example)) that I can debug.

**Comment by ghonche** [comment]
duplicate issue: https://github.com/capacitor-community/firebase-crashlytics/issues/55
